packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -5,21 +5,36 @@ import { IProcessedStyleSet } from '@fluentui/react/lib/Styling';
 import * as d3Sankey from 'd3-sankey';
 const getClassNames = classNamesFunction<ISankeyChartStyleProps, ISankeyChartStyles>();
 
-export class SankeyChartBase extends React.Component<
-  ISankeyChartProps,
-  {
-    containerWidth: number;
-    containerHeight: number;
-  }
-> {
+/*eslint-disable @typescript-eslint/no-explicit-any */
+interface ISankeyChartState {
+  containerWidth: number;
+  containerHeight: number;
+  selectedState: boolean;
+  selectedLinks: any[];
+  selectedNodes: any[];
+  selectedNode?: any;
+}
+/*eslint-disable @typescript-eslint/no-explicit-any */
+
+const restNodeOpacity: number = 1;
```

Comment:
Capitalize constants like REST_NODE_OPACITY

Reply:
I will change this